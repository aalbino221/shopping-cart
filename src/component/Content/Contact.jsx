import React from 'react';

export default function Contact() {
  return (
    <div className="contact">
      <i className="fa-brands fa-github" />
      <i className="fa-brands fa-instagram" />
      <i className="fa-brands fa-linkedin" />
    </div>
  );
}
