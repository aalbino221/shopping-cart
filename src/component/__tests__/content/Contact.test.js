import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom'; // optional
import Contact from '../../Content/Contact';

describe('Contact Component', () => {
  it('Renders Correctly', () => {
    const { container } = render(<Contact />);
    expect(container).toMatchSnapshot();
  });
});
