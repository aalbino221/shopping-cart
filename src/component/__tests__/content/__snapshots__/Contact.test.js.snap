// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Component Renders Correctly 1`] = `
<div>
  <div
    class="contact"
  >
    <i
      class="fa-brands fa-github"
    />
    <i
      class="fa-brands fa-instagram"
    />
    <i
      class="fa-brands fa-linkedin"
    />
  </div>
</div>
`;
